centreon-open-tickets/.github/workflows/dependabot_jira.yml

```diff
@@ -0,0 +1,79 @@
+name: Create Dependabot Ticket on Jira
```

Comment:
Could you delete this file ? If not, we may delete it in another PR

Reply:
done